xonsh/environ.py

```diff
@@ -926,6 +928,15 @@ def _current_job():
     return ''
 
 
+def env_name(pre_chars='(', post_chars=') '):
+    """Extract the current environment name from $VIRTUAL_ENV."""
+    env_path = builtins.__xonsh_env__.get('VIRTUAL_ENV', '')
```

Comment:
But you are applying a default value here as an empty string. It is more general to put this empty string in the default values dict, and more consistent with the rest of xonsh.  I'd be happy to make this change, if your don't want to for some reason.